app/templates/events/view.hbs

```diff
@@ -20,14 +20,16 @@
       <div class="twelve wide column {{unless device.isMobile 'right aligned'}}">
         {{#if device.isMobile}}
           <div class="ui icon fluid buttons">
-            <a href="{{href-to 'public' model.id}}" class="ui button">
+            <a href="{{href-to 'public' model.id}}" class="ui button" data-tooltip={{t 'Preview'}} data-position="top left">
               <i class="unhide icon"></i>
             </a>
-            <button class="ui button" {{action 'togglePublishState'}}>
+            <button class="ui button" {{action 'togglePublishState'}} data-tooltip="{{if (eq model.state 'published') 'Unpublish' 'Publish'}}" data-position="top left">
               <i class="{{if (eq model.state 'published') 'ban' 'check'}} icon"></i>
             </button>
-            <button class="ui button"><i class="copy icon"></i></button>
-            <button class="ui red button" {{action 'openDeleteEventModal'}}><i class="trash icon"></i></button>
+            <button class="ui button {{if isCopying 'loading'}}" {{action 'copyEvent'}} disabled="{{isCopying}}" data-tooltip={{t 'Copy'}} data-position="top left">
```

Comment:
if a button is in loadign state, it is disabled by default. No need to make it explicit.

Reply:
@CosmicCoder96 Review it